src/pages/APIs/__tests__/PredictionAPI.test.tsx

```diff
@@ -33,14 +34,18 @@ describe('PredictionAPI', () => {
 
   it('postTranslationEndpoint', () => {
     const text = 'text';
+    const languagePair = {
+      from: 'Igbo',
+      to: 'English',
+    };
```

Comment:
Our code will be more reliable if we use enums over strings in this case

```suggestion
    const languagePair = {
      from: LanguageEnum.IGBO,
      to: LanguageEnum.ENGLISH,
    };
```